tests/test_weighted_pandas.py

```diff
@@ -23,7 +61,106 @@ def test_WeightedDataFrame_constructor():
 
 def test_WeightedDataFrame_slice():
     df = test_WeightedDataFrame_constructor()
-    assert isinstance(df['A'],  WeightedSeries)
+    assert isinstance(df['A'], WeightedSeries)
     assert df[:10].shape == (10, 3)
     assert df[:10].weight.shape == (10,)
     assert df[:10]._rand.shape == (10,)
+
+
+def test_WeightedDataFrame_mean():
+    df = test_WeightedDataFrame_constructor()
+    mean = df.mean()
+    assert isinstance(mean, Series)
+    assert_allclose(mean, 0.5, atol=1e-2)
+
+
+def test_WeightedDataFrame_std():
+    df = test_WeightedDataFrame_constructor()
+    std = df.std()
+    assert isinstance(std, Series)
+    assert_allclose(std, (1./12)**0.5, atol=1e-2)
+
+
+def test_WeightedDataFrame_cov():
+    df = test_WeightedDataFrame_constructor()
+    cov = df.cov()
+    assert isinstance(cov, DataFrame)
+    assert_allclose(cov, (1./12)*numpy.identity(3), atol=1e-2)
+
+
+def test_WeightedDataFrame_median():
+    df = test_WeightedDataFrame_constructor()
+    median = df.median()
+    assert isinstance(median, Series)
+    assert_allclose(median, 0.5, atol=1e-2)
+
+
+def test_WeightedDataFrame_quantile():
+    df = test_WeightedDataFrame_constructor()
+    for q in numpy.linspace(0, 1, 10):
+        quantile = df.quantile(q)
+        assert isinstance(quantile, Series)
+        assert_allclose(quantile, q, atol=1e-2)
+
+
+def test_WeightedDataFrame_neff():
+    df = test_WeightedDataFrame_constructor()
+    neff = df.neff()
+    assert isinstance(neff, float)
+    assert neff < len(df)
+    assert neff > len(df) * numpy.exp(-0.25)
```

Comment:
Is this last line generally true? Where is the `-0.25` coming from?
If we were to reduce `len(df)` to 100 this would be false quite frequently, right?

Reply:
It's definitely more of a sanity check that neff is roughly the right amount, and it would only be true for large dataframes like the one in this case.

Reply:
ok